tests/functional/GdprCest.php

```diff
@@ -159,12 +159,29 @@ public function testPrivacyPageAccess(FunctionalTester $I)
     {
 
         $I->amGoingTo('Try that a user cant access to privacy if GDPR is not enabled');
-        $this->_prepareModule(false, false,false);
+        $this->_prepareModule(false, false, false);
         $I->amLoggedInAs(1);
         $I->amOnRoute('/user/settings/privacy');
         $I->see('Not Found');
         $I->amOnRoute('/user/settings/privacy');
-        $I->see('Not Found');$I->amOnRoute('/user/settings/privacy');
         $I->see('Not Found');
+        $I->amOnRoute('/user/settings/privacy');
+        $I->see('Not Found');
+    }
+
+    public function testForcedConsentRequirement(FunctionalTester $I)
```

Comment:
it would be nice to have another test, where an user with given consent is **not** asked for consent